spec/support/common_client_stubs.rb

```diff
@@ -0,0 +1,43 @@
+shared_context :stub_api_following_standards do
+  let(:response) do
+    {
+        'data' => [ data ].flatten,
+        'page' => 1,
+        'page_size' => 1,
+        'total' => 1
```

Comment:
I tried making that change and it causes me to have to edit 18 let(:data) statements to add in [] so I'm not sure I like it better.